Modelica/Magnetic/QuasiStatic/FluxTubes/package.mo

```diff
@@ -11,8 +11,7 @@ package FluxTubes "Library for modelling of quasi-static electromagnetic devices
 This library is intended to provide models for the investigation of
 quasi-static electromagnetic devices with lumped magnetic networks.
 </p>
-</html>"),
-           Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+</html>"), Icon(coordinateSystem(preserveAspectRatio=false), graphics={
```

Comment:
I know that this change isn't needed. But the previous variant was plain weird.

Reply:
You're right, but there is plenty of such cases in the MSL code. :-(